### ISSUE TITLE
Day 1 Deliverables

### ISSUE THREAD

**Issue by tysnj** [tysnj/front-end-news#1]
Create a DM with both instructors and send the following:
- [X] 	◦	Your niche audience (be as specific and narrow as possible)
> This app is for users who want a curated list of Hacker News posts based on front-end technologies
- [X] 	◦	Your chosen API
> https://hn.algolia.com/api
> https://github.com/HackerNews/API
- [X] 	◦	The information you will be using from it
> Pulling stories based on keywords
- [X] 	◦	A list of your app’s MVP features
> Users can save posts to a reading-list
> Users can remove posts from their reading-list
- [X] 	◦	Optional: a list of your app’s nice-to-have features
> Users can filter based on the front-end focussed tabs
- [X] 	◦	A backup API & audience
> This is my backup! If I have to, I'll narrow scope to React 18 updates.

Once your project MVP is approved (instructors will help you narrow down your MVP to a reasonable scope), please also send the following: 
>>>>>>  https://miro.com/app/board/o9J_lMhz0Dg=/?   <<<<<<<<<

- [X]	•	Wireframes
- [X]	•	Links to 2 design inspirations
- [X]	•	Project board link
- [X]	•	Repo link
- [X]	•	2 separate user personas describing people in your niche audience (can wait until after the user persona lesson)
